### DISCUSSION TITLE
Any plans to support TV platforms like Tizen and Roku?

### DISCUSSION THREAD

**Discussion by DidRanyeStudyToday** [react-native-tvos/react-native-tvos#693] [Q&A] [answered]
The introduction only mentions Apple TV and Android TV

**Comment by douglowder** [comment, marked as answer]
Those platforms will not be supported by this repo.

**Comment by hssdiv** [comment]
If somebody will find this thread in the future (searching for a way to deploy react-native-tvos to other platforms like tizen\webos) you can do this, first thing is to build your app as website (tizen\webos apps are essentially websites), I'm using expo so just used: npx expo export --platform web

Lots of other things are also needed, but it's doable
